### PR TITLE
QT1-266 RPL summary page iteration

### PR DESCRIPTION
**Pull request by alireid**

### CODE REVIEW

**Review by Pauljgraham — DISMISSED** [review]

File: src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/DraftApprenticeship/RecognisePriorLearningSummary.cshtml [hunk 1]
```diff
@@ -163,19 +144,25 @@
...
             </p>
 
-                <input type="hidden" name="answers-checked" value="true">
-                <button class="govuk-button govuk-!-margin-top-6" data-module="govuk-button">
-                    Save and Continue
-                </button>
-                <br>
-            </form>
+            <p class="govuk-body">
+                If the figures you have entered are correct your organisation needs to adjust it's calculation method to make it compliant with the funding rules.
```
Comment (chrisfoster76) [inline src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/DraftApprenticeship/RecognisePriorLearningSummary.cshtml:152]: There should be no apostrophe in `it's` (though this matches the content in the ticket).

File: src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/DraftApprenticeship/RecognisePriorLearningSummary.cshtml [hunk 2]
```diff
@@ -163,19 +144,25 @@
...
+            </p>
+
+            <input type="hidden" name="answers-checked" value="true">
+            <button class="govuk-button govuk-!-margin-top-6" data-module="govuk-button">
+                Save and Continue
```
Comment (chrisfoster76) [inline src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/DraftApprenticeship/RecognisePriorLearningSummary.cshtml:157]: Is this button label correct?

**Review by chrisfoster76 — COMMENTED** [review]
Content things

File: src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/DraftApprenticeship/RecognisePriorLearningSummary.cshtml (hunk 1 above)
Comment (alireid) [reply src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/DraftApprenticeship/RecognisePriorLearningSummary.cshtml]: Fixed Chris, thanks

**Review by chrisfoster76 — APPROVED** [review]

**Pull request merged**